graph/vertex/impl/VertexImpl.java

```diff
@@ -39,10 +38,6 @@ public void iid(VERTEX_IID iid) {
         this.iid = iid;
     }
 
-    public boolean isModified() {
-        return isModified;
-    }
-
```

Comment:
I think this should stay at `VertexImpl`, and we just override it once at `ThingVertexImpl.Read` to always return `false`.